pydcs_extensions/vietnamwarvessels/vietnamwarvessels.py

```diff
@@ -0,0 +1,1266 @@
+        AN_M57___250lb_GP_Bomb_LD = (14, Weapons.AN_M57___250lb_GP_Bomb_LD)
+        AN_M81___260lb_GP_Bomb_LD = (14, WeaponsVWV.AN_M81___260lb_GP_Bomb_LD)
+        AN_M88___220lb_GP_Bomb_LD = (14, WeaponsVWV.AN_M88___220lb_GP_Bomb_LD)
+        AN_M64___500lb_GP_Bomb_LD = (14, Weapons.AN_M64___500lb_GP_Bomb_LD)
+        Mk_82___500lb_GP_Bomb_LD = (14, Weapons.Mk_82___500lb_GP_Bomb_LD)
+        Mk_20_Rockeye___490lbs_CBU__247_x_HEAT_Bomblets = (
+            14,
+            Weapons.Mk_20_Rockeye___490lbs_CBU__247_x_HEAT_Bomblets,
+        )
+        LAU_68___7_x_UnGd_Rkts__70_mm_Hydra_70_M151_HE = (
+            14,
+            Weapons.LAU_68___7_x_UnGd_Rkts__70_mm_Hydra_70_M151_HE,
+        )
+        LAU_61___19_x_UnGd_Rkts__70_mm_Hydra_70_M156_SM = (
+            14,
+            Weapons.LAU_61___19_x_UnGd_Rkts__70_mm_Hydra_70_M156_SM,
+        )
+        _3_x_4_5_inch_M8_UnGd_Rocket = (14, Weapons._3_x_4_5_inch_M8_UnGd_Rocket)
+        SUU_25_x_8_LUU_2___Target_Marker_Flares = (
+            14,
+            Weapons.SUU_25_x_8_LUU_2___Target_Marker_Flares,
+        )
+        HVAR__UnGd_Rkt = (14, Weapons.HVAR__UnGd_Rkt)
+
+    class Pylon15:
+        AN_M30A1___100lb_GP_Bomb_LD = (15, Weapons.AN_M30A1___100lb_GP_Bomb_LD)
+        AN_M57___250lb_GP_Bomb_LD = (15, Weapons.AN_M57___250lb_GP_Bomb_LD)
+        AN_M81___260lb_GP_Bomb_LD = (15, WeaponsVWV.AN_M81___260lb_GP_Bomb_LD)
+        AN_M88___220lb_GP_Bomb_LD = (15, WeaponsVWV.AN_M88___220lb_GP_Bomb_LD)
+        AN_M64___500lb_GP_Bomb_LD = (15, Weapons.AN_M64___500lb_GP_Bomb_LD)
+        Mk_82___500lb_GP_Bomb_LD = (15, Weapons.Mk_82___500lb_GP_Bomb_LD)
+        Mk_20_Rockeye___490lbs_CBU__247_x_HEAT_Bomblets = (
+            15,
+            Weapons.Mk_20_Rockeye___490lbs_CBU__247_x_HEAT_Bomblets,
+        )
+        LAU_68___7_x_UnGd_Rkts__70_mm_Hydra_70_M151_HE = (
+            15,
+            Weapons.LAU_68___7_x_UnGd_Rkts__70_mm_Hydra_70_M151_HE,
+        )
+        LAU_61___19_x_UnGd_Rkts__70_mm_Hydra_70_M156_SM = (
+            15,
+            Weapons.LAU_61___19_x_UnGd_Rkts__70_mm_Hydra_70_M156_SM,
+        )
+        _3_x_4_5_inch_M8_UnGd_Rocket = (15, Weapons._3_x_4_5_inch_M8_UnGd_Rocket)
+        SUU_25_x_8_LUU_2___Target_Marker_Flares = (
+            15,
+            Weapons.SUU_25_x_8_LUU_2___Target_Marker_Flares,
+        )
+        HVAR__UnGd_Rkt = (15, Weapons.HVAR__UnGd_Rkt)
+
+    pylons: Set[int] = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15}
+
+    tasks = [
+        task.GroundAttack,
+        task.CAS,
+        task.AFAC,
+        task.RunwayAttack,
+        task.AntishipStrike,
+    ]
+    task_default = task.CAS
+
+
+@planemod
+class vwv_ra_5(PlaneType):
+    id = "vwv_ra-5"
+    height = 5.91
+    width = 16.16
+    length = 23.32
+    fuel_max = 10000
+    max_speed = 2469.6
+
+    livery_name = "VWV_RA-5"  # from type
+
+    pylons: Set[int] = set()
+
+    tasks = [task.Reconnaissance]
```

Comment:
This explains why you're having with CAS & Armed Recon. Does Reconnaissance allow for an "Engage targets in zone" task?